latex/Research_Strategy.tex

```diff
@@ -450,7 +497,7 @@
 As in SAMPL3~\cite{muddana_sampl3_2012}, these acyclic CB[$n$]-type receptors introduce conformational complexity, and water interactions play a key role.
 Moreover, the presence of 4 anionic groups near the cavity will likely impact the balance between ion-dipole interactions and solvation of the free host.
 
-{\bf Subaim 2.2. Gibb deep cavity cavitands for host-guest studies} 
+\textbf{Subaim 2.2. Gibb deep cavity cavitands for host-guest studies} 
```

Comment:
The octa-acids are the specific "octa acid" hosts used in SAMPL4 and SAMPL5. The GDCCs are the whole class. I can try to clarify. The point is, once you remove the acidic groups (as in some of the later SAMPLs here) it is no longer an acid, so the "octa acid" name is inappropriate. Mike Gilson and I realized this recently.